zigbee2mqtt/config.json

```diff
@@ -35,7 +47,7 @@
     "8099/tcp": "Frontend tcp-listen port"
   },
   "options": {
-    "data_path": "/config/zigbee2mqtt",
+    "data_path": "/addon_config/zigbee2mqtt",
```

Comment:
This seems to be breaking for existing installations, I've tried updating from an older version and the `data_path` changes to `/addon_config/zigbee2mqtt` which will result in a clean instance.

Reply:
Hmm I gave this a few more tests and it seems like the path does indeed get overridden if no option is ever changed, since I'm not using the default mqtt setup I've always changed that and never noticed...

Reply:
I've added an automatic migration to the entrypoint script